src/_variables.scss

```diff
@@ -42,3 +42,17 @@ $c2lc-action-panel-height: 3rem;
 $c2lc-scene-border-width: 0.1rem;
 $c2lc-scene-border-color: #CCE6E2;
 $c2lc-scene-border-style: solid;
+
+/* mixins */
+
+@mixin updatedProgramAnimation($color, $commandName, $theme) {
+    @keyframes updatedAnimation-#{$commandName}-#{$theme} {
+        0% {box-shadow: 0 0 0 1rem $color;}
+        25% {box-shadow: 0 0 0 0.75rem $color;}
```

Comment:
As mentioned in my comment with gifs, the transition seemed much smoother without the 25/50/75 percent keyframes.  Was there another reason you needed them?